### ISSUE TITLE
Efficient way to show difference between snapshots

### ISSUE THREAD

**Issue by scoddy** [restic/restic#10]
Addition to list snapshots: As a user I want to see the increments between snapshots (in files + file size).

**Comment by fw42** [comment]
What's the difference between this issue and https://github.com/restic/restic/issues/11? Sounds the same to me.

**Comment by fd0** [comment]
#11 should list the differences in detail, e.g. which files exactly have been added/removed/modified. The feature that is meant here should display statistical information. For example, after creating a new snapshot it would be nice to have some statistics on how many files were added/removed/changed and how much additional data was backed up.

**Comment by zcalusic** [comment]
I agree this would be a great feature. For example, here's what rdiff-backup reports at the end of the backup session:
```
--------------[ Session statistics ]--------------
StartTime 1476583301.00 (Sun Oct 16 04:01:41 2016)
EndTime 1476584169.19 (Sun Oct 16 04:16:09 2016)
ElapsedTime 868.19 (14 minutes 28.19 seconds)
SourceFiles 582240
SourceFileSize 25381847429 (23.6 GB)
MirrorFiles 581659
MirrorFileSize 25341511668 (23.6 GB)
NewFiles 1266
NewFileSize 145649503 (139 MB)
DeletedFiles 685
DeletedFileSize 21369080 (20.4 MB)
ChangedFiles 10234
ChangedSourceSize 931040607 (888 MB)
ChangedMirrorSize 1014985269 (968 MB)
IncrementFiles 12190
IncrementFileSize 100471118 (95.8 MB)
TotalDestinationSizeChange 140806879 (134 MB)
Errors 0
--------------------------------------------------
```
The most useful of all these statistics is TotalDestinationSizeChange, which reports how much data was added to the repository in the particular backup run. When you monitor this metric on a daily basis, you get a feeling of how much data typically changes in between backup runs. If the number suddenly jumps, it typically means that there's some new cache or unwanted folder that should be excluded.

**Comment by fd0** [comment]
Ah, this is already implemented in 0.8.1, closing.

**Comment by alphapapa** [comment]
It's quite fast too.  Works great!  I added support for it in =restic-runner= so you can easily see files that are added, changed, or removed.
